BenchmarkSamples/benchmark/build.gradle

```diff
@@ -13,12 +13,11 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
-    testBuildType "benchmark"
     buildTypes {
-        benchmark {
+        debug {
             debuggable false
             minifyEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'benchmark-proguard-rules.pro'
             signingConfig signingConfigs.debug
```

Comment:
signingConfig is no longer needed with debug{}

Reply:
:+1: